### DISCUSSION TITLE
Missing eTIV Stats in nibabies 24.1.0 mcribs Output – Need Guidance

### DISCUSSION THREAD

**Discussion by yanbin-niu** [nipreps/nibabies#413] [Q&A] [answered]
Hi!

We are using the nibabies 24.1.0 `mcribs` option to process anatomical data. However, the output folder sourcedata/freesurfer/sub-xxx/stats does not include the eTIV statistics. Typically, eTIV is found in brainvol.stats in FreeSurfer or as a separate eTIV.txt file in infant FreeSurfer.

I’m wondering if I might have missed something in the process, or if the eTIV statistics are located elsewhere under a different name or format. Any guidance would be greatly appreciated!

**Comment by mgxd** [comment, marked as answer]
Hi,

Since mcribs doesn't perform a registration to talairach space, the [`talaraich.xfm` needed for eTIV](https://github.com/DCAN-Labs/MCRIBS_for_MAKEGI/blob/b9902249edf26baf71fa4c4ba69dca3c6b578acf/bin/MCRIBSegStats#L21) is not available out-the-box 


I haven't tried supplementing this, but imo your best bet would be getting a decent talairach registration, either through [infant freesurfer's method](https://github.com/freesurfer/freesurfer/blob/8b40551f096294cc6603ce928317b8df70bce23e/infant/infant_recon_all#L743-L779) or [recon-all's](https://surfer.nmr.mgh.harvard.edu/fswiki/ReconAllDevTable), and then using `mri_segstats`

**Comment by yanbin-niu** [comment]
Aha, got it! This sounds great to me. I really appreciate your answer—it’s incredibly helpful!
